newIDE/app/src/UI/Theme/DefaultLightTheme/theme.json

```diff
@@ -153,6 +153,14 @@
         }
       }
     },
+    "icon-button": {
+      "selected": {
+        "background-color": {
+          "value": "#7046EC",
+          "comment:": "Palette/Purple/40"
```

Comment:
looking at the screenshots, I think having the same "light" purple than on the darktheme would make more sense.
To avoid making them stand out as much as the button preview/publish.

Reply:
So I'd suggest a Purple 10/20

Reply:
I followed @LuniMoon mock-up there, but happy to reconsider

Reply:
oh sorry, if there's a mock-up, then fine. I thought you made the decision yourself. :)

Reply:
I do agree with the fact that they can stand out too much (they might even look like buttons), but I am trying to stay as close as possible to the Material Primary colours as possible...
If I have your permission (to "break" that "primary colour" rule for **this** panel), I can choose a lighter purple.

Reply:
Let's try something more light then!

Reply:
Updated.
Check Figma "#2 Layout improvement" under page "Testing" for the updated colours on hover and clicks.

Reply:
Updated 👍